src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -277,6 +278,14 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
     }
   }
 
+  private fun nestedTypesSimpleNames(): Set<String> {
+    var nestedTypesSimpleNames = HashSet<String>(typeSpecs.size)
+    for (typeSpec in typeSpecs) {
+      nestedTypesSimpleNames.add(typeSpec.name)
+    }
+    return nestedTypesSimpleNames
+  }
```

Comment:
This function can be replaced by `typeSpecs.map { it.name }.toSet()`

Reply:
Done :)